docs/src/pages/Home/HomePage.js

```diff
@@ -143,22 +142,19 @@ const HomePage = (props, context) => {
 
   return (
     <section>
-      <Helmet title="Backpack" />
```

Comment:
Using the fallback defined above instead. This looks better than the previous value of `Backpack | Backpack`.